controllers/ibmblockcsi_controller.go

```diff
@@ -619,6 +647,7 @@ func (r *IBMBlockCSIReconciler) getClusterRoles(instance *ibmblockcsi.IBMBlockCS
 	csiAddonsReplicator := instance.GenerateCSIAddonsReplicatorClusterRole()
 	controllerSCC := instance.GenerateSCCForControllerClusterRole()
 	nodeSCC := instance.GenerateSCCForNodeClusterRole()
+	callHomeSCC := instance.GenerateSCCForCallHomeClusterRole()
```

Comment:
could you open a ticket to fix naming like these? SCC is not a cluster role

Reply:
CSI-3930